toolchain/check/context.cpp

```diff
@@ -345,6 +451,17 @@ auto Context::LookupQualifiedName(Parse::NodeId node_id, SemIR::NameId name_id,
         scopes.push_back({.name_scope_id = extended_id,
                           .specific_id = SemIR::SpecificId::Invalid});
       }
+      is_parent_access |= !extended.empty();
+
+      // If this is the last scope and we still haven't found a valid
+      // lookup, diagnose an invalid access.
+      if (scopes.empty() && is_access_not_allowed) {
+        DiagnoseInvalidAccess(node_id, /*member_id*/ scope_result_id, name_id,
```

Comment:
```suggestion
        DiagnoseInvalidAccess(node_id, /*member_id=*/scope_result_id, name_id,
```

I note this elsewhere too, but the `=` is significant for clang-format.